target/scripts/helpers/log.sh

```diff
@@ -116,4 +108,32 @@ function _log
   fi
 }
 
-function _log_with_date { _log "${1}" "$(date '+%Y-%m-%d %H:%M:%S')  ${2}" ; }
+# Like `_log` but adds a timestamp in front of the message.
+function _log_with_date
+{
+  _log "${1}" "$(date '+%Y-%m-%d %H:%M:%S')  ${2}"
+}
+
+# Get the value of the environment variable LOG_LEVEL if
+# it is set. Otherwise, try to query the common environment
+# variables file. If this does not yield a value either,
+# use the default log level.
+function _get_log_level_or_default
+{
+  if [[ -n ${LOG_LEVEL+set} ]]
+  then
+    printf '%s' "${LOG_LEVEL}"
+  elif [[ -e /etc/dms-settings ]]
+  then
+    grep "^LOG_LEVEL=" /etc/dms-settings | cut -d "'" -f 2
+  else
+    printf 'info'
```

Comment:
🤷 

```suggestion
    echo 'info'
```

Reply:
Will do in a follow-up PR, so I can merge this first :)